src/App.vue

```diff
@@ -14,7 +14,8 @@ export default {
   components: { TheHeader },
   data() {
     return {
-      savedParks: []
+      savedParks: [],
+      geolocation: ''
```

Comment:
Thanks for getting `geolocation` in App.

Reply:
I'm excited to see this in action with our new endpoint